src/main/asciidoc/http.adoc

```diff
@@ -1082,9 +1082,10 @@ Alternatively you can just parse the `Set-Cookie` headers yourself in the respon
 
 ==== 30x redirection handling
 
-The client can be configured to follow HTTP redirections: when the client receives an
-`301`, `302`, `303` or `307` status code, it follows the redirection provided by the `Location` response header
-and the response handler is passed the redirected response instead of the original response.
+The client can be configured to follow HTTP redirections provided by the Location response header when the client receives:
```

Comment:
should be \`Location\` , \`301\`, etc...

Reply:
Sorry, I was careless.